src/services/indexer/dao/mappers/proposal/types.ts

```diff
@@ -54,6 +54,10 @@ export abstract class Proposal {
   proposer: string;
   period: number;
   proposerFrozenTokens: string;
+  dao_type!: {
+    id: number;
+    name: "treasury" | "registry";
+  };
```

Comment:
Why does the proposal need to be aware of its parent DAO type? And how does the DAO type affect the proposal class?